src/main/kotlin/com/exactpro/th2/infraoperator/model/box/mq/factory/MessageRouterConfigFactoryBox.kt

```diff
@@ -19,8 +19,9 @@ package com.exactpro.th2.infraoperator.model.box.mq.factory
 import com.exactpro.th2.infraoperator.configuration.ConfigLoader
 import com.exactpro.th2.infraoperator.model.box.mq.MessageRouterConfiguration
 import com.exactpro.th2.infraoperator.model.box.mq.QueueConfiguration
-import com.exactpro.th2.infraoperator.operator.StoreHelmTh2Op.EVENT_STORAGE_PIN_ALIAS
+import com.exactpro.th2.infraoperator.operator.impl.EstoreHelmTh2Op.EVENT_STORAGE_PIN_ALIAS
```

Comment:
note: it looks a bit strange that we take a constant from some particular implementation. Probably, the constant should be moved to some common place instead

Reply:
Why common place should know about specific implementation?

Reply:
Is it a constant for specific implementation? If we replace `EstoreHelmTh2Op` with something else the pin name will still be the same I believe. But if you would prefer to keep it as is - I have no objection. Just looks a bit odd having `impl` in the imports